app/assets/stylesheets/research-plan.scss

```diff
@@ -197,4 +197,3 @@
         display: none
     }
 }
-
```

Comment:
You have removed many classes in the components.
Are there any classes left from this file?
If not, you could delete this file